src/webui/api/synccontroller.cpp

```diff
@@ -497,6 +497,7 @@ void SyncController::updateFreeDiskSpace(const qint64 freeDiskSpace)
 //  - "max_seeding_time": Upload max seeding time
 //  - "ratio_limit": Upload share ratio limit
 //  - "seeding_time_limit": Upload seeding time limit
+//  - "share_limit_action": Action to execute when the limit is reached
```

Comment:
Note that WebAPI changes should be listed in [WebAPI Changelog](https://github.com/qbittorrent/qBittorrent/blob/master/WebAPI_Changelog.md).